v3/setup/klam-ssh.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash -xe
+
+etcdctl set /images/klam-ssh  "adobecloudops/klam-ssh:latest"
+
+AZ=$(curl -s http://169.254.169.254/latest/meta-data/placement/availability-zone)
+REGION=${AZ::-1}
+ROLE_NAME="$(etcdctl get /klam-ssh/config/role-name)"
```

Comment:
If the KLAM Role is created in the infrastructure stack, it should be added as an environment variable here: https://github.com/adobe-platform/infrastructure/blob/master/v2/src/userdata/templates/write_files/environment/shared.yml

Otherwise, we will have to have users pull their KLAM role out of the infrastructure stack and insert it into secrets.